x/merkledb/codec_test.go

```diff
@@ -219,18 +217,18 @@ func FuzzEncodeHashValues(f *testing.F) {
 			randSeed int,
 		) {
 			require := require.New(t)
-			for _, branchFactor := range branchFactors { // Create a random *hashValues
+			for _, tokenConfig := range validTokenConfigurations { // Create a random *hashValues
```

Comment:
We should probably remove this reference (and others) to `hashValues` since that type is gone